quirks.bs

```diff
@@ -35,10 +35,12 @@ urlPrefix: https://dbaron.org/css/intrinsic/
 </pre>
 
 <pre class=link-defaults>
-spec:css21; type:property; text:min-width
-spec:css21; type:property; text:min-height
-spec:css21; type:property; text:max-width
-spec:css21; type:property; text:max-height
+spec:css2; type:property; text:min-width
+spec:css2; type:property; text:min-height
+spec:css2; type:property; text:max-width
+spec:css2; type:property; text:max-height
+spec:css-sizing-3; type:property; text:box-sizing
+spec:css-sizing-3; type:value; text:border-box
```

Comment:
@tabatkins I still get
```
\033[1;33mLINK ERROR:\033[0m Multiple possible 'border-box' maybe refs for 'box-sizing'.
Arbitrarily chose https://drafts.csswg.org/css-sizing-3/#valdef-box-sizing-border-box
To auto-select one of the following refs, insert one of these lines into a &lt;pre class=link-defaults> block:
spec:css-ui-3; type:value; text:border-box
spec:css-sizing-3; type:value; text:border-box
''box-sizing/border-box''
```
which does not make much sense as far as I can tell?

Reply:
Oh, indeed, and I bet I know why. I'll look into this, but for now, you can fix it by adding a `for:box-sizing` to the line. Sorry for the bad error message.